### PR TITLE
Feature/parse time estimates

### PR DESCRIPTION
**Pull request by learykara**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,10 @@
 {
   "name": "@thinkful/zapdos",
-  "description": "Content and program structure build tool for Thinkfu",
-  "version": "1.0.1",
-  "author": "ripleyaffect @ripleyaffect",
```
Comment (learykara) [inline package.json]: sorry, man

Comment (ripleyaffect) [reply package.json]: :sob:

File: src/lib/validateCheckpoint.js [hunk 2]
```diff
@@ -26,6 +29,18 @@ module.exports = async (checkpoint, checkpoints = null) => {
     errors.push(error);
   }
 
+  // Check for missing time estimate
+  if (typeof checkpoint.attributes.time !== 'number') {
+    const error =
+      `CHECKPOINT_INVALID: Checkpoint ${checkpoint.src} ` +
+      `has a missing or invalid time estimate. ` +
+      `Times must be provided in hours or minutes.`;
```
Comment (ripleyaffect) [inline src/lib/validateCheckpoint.js]: may want to log the value here and be more explicit about the format. Previously, time estimates could be numbers and, based on their range, the tool would guess if it was mins or hrs, so some folks may be used to putting numbers and not realize they need to explicitly have `(hour|minute)s` in there

File: src/lib/getTimeEstimateHours.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+const MINUTES_PER_HOUR = 60;
+
+module.exports = checkpoint => {
+  const {
+    attributes: { time },
+  } = checkpoint;
```
Comment (ripleyaffect) [inline src/lib/getTimeEstimateHours.js]: can be destructured in the function signature

**Pull request merged**